_posts/2018-03-25-this-week-in-rustfest-3-train-strike-cfp-keynote-speakers.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "These weeks in RustFest 3: Train Strike, CfP Progress & Keynote Speakers"
```

Comment:
I believe it should be "This week in RustFest 3..." (but better check with someone sure, or if you are sure)

Reply:
We did that last time because we missed the weekly window a bit.

Reply:
Ok